internal/cli/atlas/accesslists/create.go

```diff
@@ -80,26 +83,81 @@ func (opts *CreateOpts) newProjectIPAccessList() []*atlas.ProjectIPAccessList {
 	return []*atlas.ProjectIPAccessList{entry}
 }
 
+func iPAddress() (string, error) {
+	if publicIP := store.IPAddress(); publicIP != "" {
+		return publicIP, nil
+	}
+
+	err := fmt.Errorf(currentIPAddressNotFound)
+	return "", err
```

Comment:
[nit] in go prefer simplicity and readability, nothing wrong with inlining things and usually preferred 
```suggestion
	return "", fmt.Errorf("not able to find your public IP address. Please providing the desired IP address for this command")
```